docs/_static/nav-config.js

```diff
@@ -0,0 +1,13 @@
+const navConfig = {
+    "navTextLinks": [
+
```

Comment:
No `"title"` or `"href"` ?

Reply:
Not sure what we should have in the navbar. Any ideas what to put here?